code/game/gamemodes/cm_process.dm

```diff
@@ -170,6 +170,41 @@ dat += " You failed to evacuate \the [MAIN_SHIP_NAME]"
 		to_chat(world, dat)
 
 
+/datum/game_mode/proc/declare_completion_announce_round_stats()
```

Comment:
this has nothing to do with xenomorph abilities, please split it into a separate PR